spec/requests/healthcheck_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+RSpec.describe "Healthcheck" do
+  describe "GET /healthcheck" do
+    before do
+      populate_default_government_bulk_data
+    end
+
+    it "returns a 200 HTTP status" do
+      get healthcheck_path
+      expect(response.status).to eq(200)
```

Comment:
minor, we've done this elsewhere with `expect(response).to have_http_status(:ok)`

Reply:
Nice, need to get better with native rspec matchers. Have adjusted.